api/core/television/television.sendCommand.js

```diff
@@ -0,0 +1,30 @@
+const Promise = require('bluebird');
+const queries = require('./television.queries.js');
+
+module.exports = function sendCommand(functionName, params) {
+    return getDeviceType(params)
+      .then((deviceType) => {
+
+          if(!gladys.modules[deviceType.service] || !gladys.modules[deviceType.service].television)
+                return Promise.reject(new Error(`television : Module ${deviceType.service} does not exist or does not handle television.`));
+         
+          if(typeof gladys.modules[deviceType.service].television[functionName] != 'function') {
+            console.log(`television : Module ${deviceType.service} does not have function ${functionName}`)
+            return Promise.reject();
```

Comment:
reject with a new Error(`television : Module ${deviceType.service} does not have function ${functionName}`);

Reply:
Fix this on commit d593238